src/hooks/useSwap.ts

```diff
@@ -77,15 +74,6 @@ export const useSwap = ({ accountId, selector }: Props) => {
   const { getNearBlock } = useNearBlock()
   const { getTransactionScan } = useTransactionScan()
 
-  const handleError = (e: unknown) => {
```

Comment:
The utility hook , should not concern itself with the display logic of an error. It introduces unnecessary complexity.